package.nls.json

```diff
@@ -244,5 +244,7 @@
     "vscode-docker.views.dockerRegistries": "Registries",
     "vscode-docker.views.dockerNetworks": "Networks",
     "vscode-docker.views.dockerVolumes": "Volumes",
-    "vscode-docker.views.activityBar": "Docker"
+    "vscode-docker.views.activityBar": "Docker",
+    "vscode-docker.utils.dockerode.sshAgent": "In order to use an SSH DOCKER_HOST, you must configure an ssh-agent.",
```

Comment:
Strings only need to be in package.nls.json if they appear in package.json. The rest of the loc stuff happens by some other magic.

Reply:
Oops, didn't know that. Fixed.